common/supplemental/supplementalData.xml

```diff
@@ -4535,8 +4535,8 @@ XXX Code for transations where no currency is involved
         <calendar type="gregorian">
            <calendarSystem type="solar" />
            <eras>
-               <era type="0" end="0-12-31" code="gregory-inverse" aliases="bc bce"/>
-               <era type="1" start="1-01-01" code="gregory" aliases="ad ce"/>
+               <era type="0" end="0-12-31" code="bce" aliases="bc"/> <!-- Before Common Era, Before Christ -->
+               <era type="1" start="1-01-01" code="ce" aliases="ad" primary="true"/> <!-- Common Era, Anno Domini -->
```

Comment:
what does `primary` mean, and when is it ever not the last era?

Reply:
I haven't written the spec yet but `primary` means "the era used for extended year"; it is `ce` in Japanese which is not the last era

Reply:
makes sense.